events.html

```diff
@@ -0,0 +1,106 @@
+                    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor03" aria-controls="navbarColor03" aria-expanded="false" aria-label="Toggle navigation">
+                        <span class="navbar-toggler-icon"></span>
+                    </button>
+
+                    <div class="collapse navbar-collapse" id="navbarColor03">
+                        <ul class="navbar-nav ml-auto">
+                            <li class="nav-item">
+                                <a class="nav-link" href="docs/index.html ">
+                                    <i class="fas fa-book fa-fw "></i>
+                                    Docs
+                                </a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="https://pypi.python.org/pypi/c7n ">
+                                    <i class="fas fa-cloud-download-alt fa-fw "></i>
+                                    Install
+                                </a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="https://github.com/cloud-custodian/cloud-custodian ">
+                                    <i class="fab fa-sm fa-github"></i>
+                                    GitHub
+                                </a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="https://gitter.im/cloud-custodian/cloud-custodian ">
+                                    <i class="fab fa-sm fa-gitter"></i>
+                                    Community
+                                </a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="events.html">
+                                    <i class="fa fa-calendar"></i>
+                                    Calendar
+                                </a>
+                            </li>
+                        </ul>
+                    </div>
+                </nav>
+            </header>
+
+            <main style="padding-top: 5em">
+        
+                <section>
+                    <iframe
+                    src="https://calendar.google.com/calendar/embed?height=600&wkst=1&bgcolor=%23ffffff&ctz=America%2FDetroit&src=Y19naDhzc2pia3JvaWVxcWFzN2trbWgzdG9yc0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t&color=%23616161&title=Cloud%20Custodian"
+                    style="border:solid 1px #777;  display:block;" width="800" height="600"
+                    scrolling="no"></iframe>
+                </section>
+            <section>
+                <strong>Contact jorge@stacklet.io if you'd like to add a Cloud Custodian
+                related event to this community calendar.</strong>
+            </section>
+        </main>
```

Comment:
Major nit: but spacing <3

```suggestion
                <section>
                    <strong>Contact jorge@stacklet.io if you'd like to add a Cloud Custodian
                    related event to this community calendar.</strong>
                </section>
            </main>
```